src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1377,6 +1377,14 @@ private function registerLockConfiguration(array $config, ContainerBuilder $cont
                     case 'flock' === $storeDsn:
                         $storeDefinition = new Reference('lock.store.flock');
                         break;
+                    case 0 === strpos($storeDsn, 'flock'):
+                        $flockPath = substr($storeDsn, 8);
+                        
+                        $storeDefinitionId = 'lock.flock.store.'.$container->hash($storeDsn);
```

Comment:
`'.lock.flock.store.'.$container->hash($storeDsn);` as per @nicolas-grekas 's comment

Reply:
Can someone enlighten me about purpose of the starting dot?

Reply:
see https://github.com/symfony/symfony/pull/26923/files/dae20b46a3a1515af3ba907882b828d9778b2224#r181552130 related to https://github.com/symfony/symfony/pull/26921